api/apps.py

```diff
@@ -107,8 +104,11 @@ def post(self):
                 jsonschema.validate(app_meta, APP_SCHEMA)
             except (ValueError, jsonschema.ValidationError) as e:
                 self.abort(400, str(e))
-            util.insert_app(self.app.db, app_temp, apps_path, app_meta=app_meta)  # pass meta info, prevent re-reading
+            insert_app(self.app.db, app_temp, apps_path, app_meta=app_meta)  # pass meta info, prevent re-reading
             log.debug('Recieved App: %s' % app_meta.get('_id'))
+        """
+        # XXX util.insert_app doesn't exist...?
+        raise NotImplementedError
```

Comment:
BTW just wanted to point this out -- this method made use of a nonexistent `util.insert_app` function, so I don't think it is ever actually called. Should I just delete it? Right now I hackishly "commented" it out by making the code a big string.